frontend/packages/operator-lifecycle-manager/integration-tests/scenarios/descriptors.scenario.ts

```diff
@@ -109,10 +109,7 @@ describe('Using OLM descriptor components', () => {
       }
     });
     $$('.co-create-operand__form-group').each(async (input) => {
-      await browser
-        .actions()
-        .mouseMove(input)
-        .perform();
+      browser.executeScript('arguments[0].scrollIntoView();', input);
```

Comment:
I have seen this done in the past FWIW - [packages/dev-console/integration-tests/tests/git-import-flow.scenario.ts#L47](https://github.com/openshift/console/blob/master/frontend/packages/dev-console/integration-tests/tests/git-import-flow.scenario.ts#L47)

May be able to co-locate the [scroll code](https://github.com/openshift/console/blob/master/frontend/packages/dev-console/integration-tests/utils/page.ts) in the root test code or something.

Reply:
Yep, definitely a reusable piece of code. But would you want it in this PR?

Reply:
In my opinion, I think it would be helpful for code reuse rather than doing the above line. Easy to move the file to say `frontend/integration-tests/utils/` (new folder), update the one import reference and then use it here.

It seems like good reusable code but I'm not looking to bog down your PR with unnecessary changes.

Reply:
Both I think do the net same thing, as `arguments[0]` is just use the property from the 2nd argument. It's just the one I linked to looks nicer. I suspect no difference between the two.

Reply:
> May be able to co-locate the scroll code in the root test code or something.

@andrewballantyne does that scroll helper need an `await`?

I agree we should generalize it for reuse outside of dev console

Reply:
I think we'd want an `await` here as well. (But I'd prefer to go ahead and reuse the existing function to avoid duplication.)

Reply:
This module seems like a good place for it:

https://github.com/openshift/console/blob/master/frontend/packages/console-shared/src/test-utils/utils.ts

Reply:
Hmm, quite possibly @spadgett I bet it's not synchronous, but I don't know for sure. It seemed to work without issue during my tests on the linked PR above -- I'll be honest I wasn't thinking about the asynchronous nature and more of the "why do we need it" for that case.

Reply:
> Hmm, quite possibly @spadgett I bet it's not synchronous, but I don't know for sure. It seemed to work without issue during my tests on the PR the code was added in

It returns a promise, so we should add an `await` to make sure it's done. Otherwise we're racing.

https://www.protractortest.org/#/api?view=webdriver.WebDriver.prototype.executeScript

> I'll be honest though, I wasn't thinking about the asynchronous nature and more of the "why do we need it" for that case.

Yeah, that I'm not sure about :/